Hotel/Hotel.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hotel
+{
+    internal class Hotel
+    {
+        List<Room> rooms = new List<Room>();
+        DateTime currentDate = DateTime.Now;
+         
+        public Hotel()
+        {
+            AddRoom();
+            AddDay();
+        }
+        public void AddDay()
+        {
+            currentDate = currentDate.AddDays(1);
+        }
+        public void NumberOffRooms()
+        {
+            Console.WriteLine("В гостинице :" + rooms.Count + " номеров.");
+            foreach(var room in rooms)
+            {
+                if(room.IdRoom == 1)
+                {
+                    Console.WriteLine("Oднокомнатные: "+ room.NumberRoom);
+                }
+                
+                if (room.IdRoom == 2)
+                {
+                    Console.WriteLine("Двухкомнатные: " + room.NumberRoom);
+                }                
+                if (room.IdRoom == 3)
+                {
+                    Console.WriteLine("Трехкомнатные: " + room.NumberRoom);
+                }
+            }
+
+        }
+
+        public void RoomFree()
+        {
+            UnloadingNumbers();
+
+            foreach (var room in rooms)
+            {
+                if (room.Reservation == true)
+                {
+                    Console.WriteLine("Свободные номера: " + room.NumberRoom);
+                }
+            }
+        }
+
+        public void RoomNotFree()
+        {
+            foreach (var room in rooms)
+            {
+                if (room.Reservation == false)
```

Comment:
Instead of (room.Reservation == false) you can use LOGIC NOT"!" if (!room.Reservation) in the If statement.